react-effects/my-app/src/List.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars -- REMOVE ME */
+import { useEffect, useState } from 'react';
+import readItems from './read';
+
+export default function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState([]);
+  const [error, setError] = useState();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+  useEffect(() => {
+    const possibleItems = readItems();
+    possibleItems.then(
```

Comment:
You can certainly do it with Promise.then syntax, but I think you should learn how to do it with async/await. The function parameter to useEffect is not allowed to be async, so you should declare an async function in here, and then call it.